cunumeric/utils.py

```diff
@@ -41,6 +41,14 @@
 ]
 
 
+def _broadcast_shapes(
+    *args: Union[NdShapeLike, Sequence[NdShapeLike]]
+) -> NdShape:
+    # Call _broadcast_shapes' for now.
+    # We will have a new implementation later
+    return np.broadcast_shapes(*args)  # type: ignore [arg-type]
```

Comment:
The comment seems out of sync with the code? (are calling `np.broadcast_shapes` here, not `_broadcast_shapes`)